doc/modules/tree.rst

```diff
@@ -418,39 +421,39 @@ scikit-learn uses an optimised version of the CART algorithm.
 Mathematical formulation
 ========================
 
-Given training vectors :math:`x_i \in R^n`, i=1,..., l and a label vector
-:math:`y \in R^l`, a decision tree recursively partitions the space such
-that the samples with the same labels are grouped together.
+Given training vectors :math:`x_i \in \mathbb R^n`, i=1,..., l and a label
+vector :math:`y \in \mathbb R^l`, a decision tree recursively partitions the
+space such that the samples with the same labels are grouped together.
 
 Let the data at node :math:`m` be represented by :math:`Q`. For
 each candidate split :math:`\theta = (j, t_m)` consisting of a
 feature :math:`j` and threshold :math:`t_m`, partition the data into
-:math:`Q_{left}(\theta)` and :math:`Q_{right}(\theta)` subsets
+:math:`Q_\textrm{left}(\theta)` and :math:`Q_\textrm{right}(\theta)` subsets
 
 .. math::
 
-    Q_{left}(\theta) = {(x, y) | x_j <= t_m}
+    Q_\textrm{left}(\theta) = {(x, y) | x_j <= t_m}
 
-    Q_{right}(\theta) = Q \setminus Q_{left}(\theta)
+    Q_\textrm{right}(\theta) = Q \setminus Q_\textrm{left}(\theta)
 
 The impurity at :math:`m` is computed using an impurity function
 :math:`H()`, the choice of which depends on the task being solved
 (classification or regression)
 
 .. math::
 
-   G(Q, \theta) = \frac{n_{left}}{N_m} H(Q_{left}(\theta))
-   + \frac{n_{right}}{N_m} H(Q_{right}(\theta))
+   G(Q, \theta) = \frac{n_\textrm{left}}{N_m} H(Q_\textrm{left}(\theta))
+   + \frac{n_\textrm{right}}{N_m} H(Q_\textrm{right}(\theta))
 
 Select the parameters that minimises the impurity
 
 .. math::
 
-    \theta^* = \operatorname{argmin}_\theta  G(Q, \theta)
+    \theta^* = \arg\min_\theta G(Q, \theta)
```

Comment:
I'm never sure which way I prefer with the rendering, but I guess it's a bit a matter of taste.